Dima/netflix-roulette/src/Components/Sort/Sort.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import Button from '../../Shared/Buttons/Button'
+import { parse } from 'query-string'
+import * as H from 'history'
+
+interface ISortProps {
+  handleSearchChange( sortBy: any ): void,
+  numberOfFilms: number,
+  location: H.Location
+};
+
+interface ISortState {
+  sortBy: string
+};
+
+export default class Sort extends React.Component<ISortProps, ISortState>{
+  state: ISortState = {
+    sortBy: 'relise date'
+  };
+
+  componentDidMount() {
+    const URLData = parse(this.props.location.search) as {
+      sortBy: string;
+    };
+    let { sortBy } = URLData;
+    console.log(sortBy);
+    sortBy ? this.setState({ sortBy: sortBy }) : sortBy = 'relise date';
```

Comment:
what is the purpose of setting sortBy = 'relise date'; to local variable that you don't use  ?